test/unit-test/test_lib/test_analysis/test_user.py

```diff
@@ -16,9 +16,9 @@
 class UserTest(unittest.TestCase):
```

Comment:
The variable on line-14 should move to setUp() function and assigned to null in tearDown() function.